azure_arc_servers_jumpstart/docs/onboard_server_win.md

```diff
@@ -37,21 +37,21 @@ The following README will guide you on how to connect an Windows machine to Azur
 
 ![](../img/onboard_server_win/01.png)
 
-* Download the [az_connect_win](../scripts/az_connect_win.ps1) Powershell script.
```

Comment:
We are aligned with the official marketing name
https://docs.microsoft.com/en-us/powershell/scripting/overview?view=powershell-7.1